CHANGES.rst

```diff
@@ -20,6 +20,9 @@ https://github.com/zopefoundation/Zope/blob/4.x/CHANGES.rst
   ``rfc850_date``, and ``rfc1123_date`` which used to be in ``App.Common``
   keeping backwards-compatibility imports in place.
 
+- With the exception of ``field2bytes``, field converters should not handle
+  files (`#558 <https://github.com/zopefoundation/Zope/issues/558>`_)
```

Comment:
Without reading the diff it was not clear to me what has changed. `should not` is not really expressive. I'd expect something like "no longer try to handle file like objects special".

I think this is a even a breaking change and should be marked as such here. (Usually we use a headline for breaking changes.) It should be enough to increase the version number to 5.2.

Reply:
Thank you for the suggestions which I just applied.